### PR TITLE
Fixed Issue #158: Invalid direction for slo_class

### PR DESCRIPTION
**Pull request by khansaad**
Signed-off-by: Saad Khan <saakhan@redhat.com>

### CODE REVIEW

File: src/main/java/com/autotune/analyzer/k8sObjects/ValidateAutotuneObject.java [hunk 1]
```diff
@@ -68,6 +68,11 @@ public static StringBuilder validate(HashMap<String, Object> map) {
 			errorString.append(AnalyzerErrorConstants.AutotuneObjectErrors.DIRECTION_NOT_SUPPORTED);
 		}
 
+		//check if slo_class is 'response_time' and direction is minimize
+		if(sloInfo.getSloClass().equalsIgnoreCase("response_time") && sloInfo.getDirection().equalsIgnoreCase("maximize")){
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/k8sObjects/ValidateAutotuneObject.java:72]: Please make sure spaces after `if` and before bracket `{`
```
if (...) {
}
```
I'd prefer if we check for `response_time` and `!minimize` throw error. Similarly, `throughput` and `!maximize` throw error

Comment (khansaad) [reply src/main/java/com/autotune/analyzer/k8sObjects/ValidateAutotuneObject.java]: Updated it now.

Comment (dinogun) [reply src/main/java/com/autotune/analyzer/k8sObjects/ValidateAutotuneObject.java]: I still dont see a space after if, `if (`

Comment (khansaad) [reply src/main/java/com/autotune/analyzer/k8sObjects/ValidateAutotuneObject.java]: Updated space between 'if' and brackets

**Review by dinogun — APPROVED** [review]
LGTM

**Review by chandrams — APPROVED** [review]
LGTM

**Pull request merged**